certified-connectors/Mural/README.md

```diff
@@ -0,0 +1,25 @@
+# Mural Connector
```

Comment:
@dmytroIvanovDev Please go thru the [template provided for readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) file and add required information.